.github/workflows/wheels.yml

```diff
@@ -214,8 +221,7 @@ jobs:
 
       - name: Install tables on ${{ matrix.python-version }}
         run: |
-          python -m pip install -r wheelhouse/artifact/requirements.txt
-          python -m pip install --no-index --find-links wheelhouse/artifact/ tables
+          python -m pip install --pre --find-links wheelhouse/artifact/ tables
```

Comment:
Good point.  Just curious, which is the package that we need a pre-release?

Reply:
`tables`, for all the runs except tags.
The problem here was that `numexpr` is no longer listed in `requirements.txt` that now is more for build dependencies (according to the comment in the first line). Probably we should use a more clear naming.